### ISSUE TITLE
for help ?

### ISSUE THREAD

**Issue by zhao777ying** [fengpeiyuan/flumeng-plugins-redis#1]
I want to use ycsb on redis cluster, but i had this Exception:

redis.clients.jedis.exceptions.JedisMovedDataException: MOVED 9476 192.168.1.54:
        at redis.clients.jedis.Protocol.processError(Protocol.java:104)
        at redis.clients.jedis.Protocol.process(Protocol.java:138)
        at redis.clients.jedis.Protocol.read(Protocol.java:192)
        at redis.clients.jedis.Connection.readProtocolWithCheckingBroken(Connect
        at redis.clients.jedis.Connection.getBinaryMultiBulkReply(Connection.jav
        at redis.clients.jedis.Jedis.hgetAll(Jedis.java:775)
        at com.yahoo.ycsb.db.RedisClient.read(RedisClient.java:76)
        at com.yahoo.ycsb.DBWrapper.read(DBWrapper.java:90)
        at com.yahoo.ycsb.workloads.CoreWorkload.doTransactionRead(CoreWorkload.
        at com.yahoo.ycsb.workloads.CoreWorkload.doTransaction(CoreWorkload.java
        at com.yahoo.ycsb.ClientThread.run(Client.java:241)

maybe something wrong with the jedis.client .....I try to use the redisson, but i can't change the 
code for ycsb, can you help me?
thanks!

**Comment by fengpeiyuan** [comment]
yeah, it is easy, download code and implement interface like me(https://github.com/fengpeiyuan/YCSB-plugin), then move jar to the path and run
